doc/notebooks/graphs-ipids.ipynb

```diff
@@ -579,7 +579,7 @@
      "cell_type": "code",
      "collapsed": false,
      "input": [
-      "ans.multiplot(lambda (p, q): (q[IP].src, q[IP].id))"
+      "ans.multiplot(lambda p_q: (p_q[1][IP].src, p_q[1][IP].id))"
```

Comment:
idem for all lambda tricks

Reply:
Yep, but I did not touch "Scapy in 15 minutes.ipynb" because a potential conflict/collision with your changes from #1286.

Reply:
I'v just updated also "Scapy in 15 minutes.ipynb"